common/api-review/vertexai.api.md

```diff
@@ -429,6 +432,148 @@ export enum HarmSeverity {
     HARM_SEVERITY_NEGLIGIBLE = "HARM_SEVERITY_NEGLIGIBLE"
 }
 
+// @public (undocumented)
+export enum ImagenAspectRatio {
+    // (undocumented)
+    CLASSIC_LANDSCAPE = "4:3",
+    // (undocumented)
+    CLASSIC_PORTRAIT = "3:4",
+    // (undocumented)
+    PORTRAIT = "9:16",
+    // (undocumented)
+    SQUARE = "1:1",
+    // (undocumented)
+    WIDESCREEN = "16:9"
+}
+
+// Warning: (ae-incompatible-release-tags) The symbol "ImagenGCSImage" is marked as @public, but its signature references "ImagenImage" which is marked as @internal
+//
+// @public
+export interface ImagenGCSImage extends ImagenImage {
+    gcsURI: string;
```

Comment:
I know this is a preliminary PR but before release you should resolve all the warnings in here. For warnings like this, where a public type extends a private type, it's going to cause a problem because the public docs are going to show that this interface has "gcsURI" but won't show that it has "mimeType" (the private extended property). If you also export ImagenImage, the ImagenGCSImage doc page still won't show mimeType but it will have a live link to ImagenImage where the user can then see it.

2 options: (1) make ImagenImage public, or (2) give up the extends and duplicate mimeType in an independent public interface

I'm not sure if there's any other ways around it. Here's an issue about the inheritance part specifically if that leads anywhere https://github.com/microsoft/rushstack/issues/3429

Reply:
I'll make sure to resolve all these warnings before the official PR.

I might be able to remove the top-level `ImagenImage`, so I'll try that. Thanks for sharing the issue! I know `prune-dts` will merge these types, but I guess that runs after the docs are generated so it doesn't help.